src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -67,6 +67,20 @@ static bool is_vector_shuffle(ciKlass* klass) {
   return klass->is_subclass_of(ciEnv::current()->vector_VectorShuffle_klass());
 }
 
+bool LibraryCallKit::arch_supports_vector_bitshuffle(int opc, int num_elem, BasicType elem_bt) {
```

Comment:
Could you please add an assertion for `opc` that it should be `Op_CompressBitsV or Op_ExpandBitsV` ?